frontend/www/js/omegaup/markdown.ts

```diff
@@ -539,6 +539,20 @@ export class Converter {
         );
       },
     );
+
+    this._converter.hooks.chain('postConversion', (text: string): string => {
+      return text.replace(/<a href="[^"]+?"/g, (match: string) => {
+        const url = match.slice(9, -1);
+        if (
+          !url.startsWith('https://omegaup') &&
+          !url.startsWith('http://omegaup')
+        ) {
+          return match + ' target="_blank" rel="noopener noreferrer"';
+        } else {
+          return match;
+        }
+      });
```

Comment:
Okay, instead of hardcoding the omegaUp URL, you can use `window.location.origin`. Additionally, you can obtain the URL using the second parameter of the replace function as follows:
```suggestion
      return text.replace(/<a href="([^"]+?)"/g, (match: string, url: string) => {
        if (url.startsWith(window.location.origin)) {
          return match;
        }
        return `${match} target="_blank" rel="noopener noreferrer"`;
      });
```